java/panama-bindings/generate-bindings.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+echo "Starting Panama FFM API bindings generation ..."
+REPODIR=$(cd $(dirname $0); cd ../../ ; pwd)
+CURDIR=$(cd $(dirname $0); pwd)
+CUDA_HOME=$(which nvcc | cut -d/ -f-4)
+TARGET_PACKAGE="com.nvidia.cuvs.internal.panama"
+
+# Try to verify that a include directory exists inside CUDA_HOME
+if [ ! -d "$CUDA_HOME/include" ];
+then
+  echo "$CUDA_HOME/include does not exist."
+  if [ -d "/usr/local/cuda/include" ];
+  then
+    echo "Setting CUDA_HOME to /usr/local/cuda"
+    CUDA_HOME=/usr/local/cuda
+  else
+    echo "Couldn't find a suitable CUDA include directory."
+    exit 1
+  fi
+fi
+
+JEXTRACT_COMMAND="jextract"
+
+if [[ `command -v jextract` == "" ]];
+then
+  JEXTRACT_DOWNLOAD_URL="https://download.java.net/java/early_access/jextract/22/6/openjdk-22-jextract+6-47_linux-x64_bin.tar.gz"
+  echo "jextract doesn't exist. Downloading it from $JEXTRACT_DOWNLOAD_URL.";
+  wget -c $JEXTRACT_DOWNLOAD_URL
+  tar -xvf openjdk-22-jextract+6-47_linux-x64_bin.tar.gz
+  JEXTRACT_COMMAND="jextract-22/bin/jextract"
```

Comment:
```suggestion
  export PATH="$(pwd)/jextract-22/bin/jextract:${PATH}"
```

Instead of tracking this `JEXTRACT_COMMAND` variable, I think it would be simpler to just place the just-downloaded `jextract` on `PATH` for the rest of this script. If you accept this suggestion, then also please remove other uses of `JEXTRACT_COMMAND` in this script, in favor of just calling `jextract`.